simapp/test_helpers.go

```diff
@@ -92,9 +93,16 @@ func SetupWithGenesisValSet(t *testing.T, valSet *tmtypes.ValidatorSet, genAccs
 	bondAmt := sdk.NewInt(1000000)
 
 	for _, val := range valSet.Validators {
+		// Currently validator requires tmcrypto.ed25519 keys, which don't support
+		// our Marshaling interfaces, so we need to pack them into our version of ed25519.
+		// There is ongonig work to add secp256k1 keys.
+		pk, err := ed25519.FromTmEd25519(val.PubKey)
+		require.NoError(t, err)
+		pkAny, err := codectypes.PackAny(pk)
```

Comment:
Since PackAny is deprecated, and we know pk is a proto.Message, could we just use NewAnyWithValue here?

Reply:
It's not deprecated - we want to use it internally only. It's deprecated for the external use.

Reply:
We are planning to move it to an internal package.